Letterbook.Core/AccountEventService.cs

```diff
@@ -16,7 +16,7 @@ public AccountEventService(IOptions<CoreOptions> options, IMessageBusAdapter mes
     {
         _options = options.Value;
         _messageBusAdapter = messageBusAdapter;
-        _channel = _messageBusAdapter.OpenChannel<Account>();
+        _channel = _messageBusAdapter.OpenChannel<Account>(nameof(ActivityEventService));
```

Comment:
This is in AccountEventService, but uses ActivityEventService. Is that intentional?